pkg/controller.v1beta2/tensorflow/job.go

```diff
@@ -41,7 +41,7 @@ func (tc *TFController) addTFJob(obj interface{}) {
 
 			status := common.JobStatus{
 				Conditions: []common.JobCondition{
-					common.JobCondition{
```

Comment:
Is this formatting change intentional?

Reply:
Yes. This `common.JobCondition` is redundant and could be remove.